src/subclass/widget.rs

```diff
@@ -34,6 +34,10 @@ pub trait WidgetImpl:
     fn button_release_event(&self, widget: &Widget, event: &mut gdk::EventButton) -> bool {
         self.parent_button_release_event(widget, event)
     }
+
+    fn can_activate_accel(&self, widget: &Widget, signal_id: u32) -> bool {
```

Comment:
That's a curious virtual method. To make proper use of that we need some more bindings for the signal API. To be able to work with signal ids (hint: this shouldn't be a plain `u32`), introspect signals, etc.

Reply:
Should I comment it out for now?

Reply:
Yeah, it needs some more work to be actually usable :)